.github/vale/styles/proselint/Cliches.yml

```diff
@@ -0,0 +1,782 @@
+extends: existence
+message: "'%s' is a cliche."
```

Comment:
This one seems pretty brutal, and probably more suited to creative writing than to documentation? Clichés are good to communicate commonplace ideas, which sometimes help to explain software :o

Reply:
Yes :D Same as above, the rules come in a "set", so they are all added when we run `vale sync`. proselint is only for suggestions right now. 

I hope in the long run, we can establish a few rules as "hard" ones and the rest can be used to get a bit input, same way we can use `alex` over in coral, but it's not an established rule.